src/components/App.js

```diff
@@ -1,16 +1,68 @@
-import React from "react";
-import SushiContainer from "./SushiContainer";
-import Table from "./Table";
-
-const API = "http://localhost:3001/sushis";
+import React, { useState, useEffect } from "react"
+import SushiContainer from "./SushiContainer"
+import Table from "./Table"
+import { getSushi } from "../utils/fetchers"
 
 function App() {
+  const [sushiList, setSushi] = useState(null)
+  const [moneySpent, setMoneySpent] = useState(0)
+  const [sushiWallet, setSushiWallet] = useState(100)
+
+  useEffect(() => {
+    getSushi().then((sushiResp) => {
+      const updatedSushi = sushiResp.map((sushi) => {
+        return {
+          ...sushi,
+          isEaten: false,
+        }
+      })
+      setSushi(updatedSushi)
+    })
```

Comment:
Nice solution. You could move this anonymous function into it's own function, then call it here to adhere better to single responsibly. Something like `formatSushi`. Just a thought!